src/System.Management.Automation/engine/remoting/common/RemoteSessionNamedPipe.cs

```diff
@@ -274,19 +274,6 @@ internal static SECURITY_ATTRIBUTES GetSecurityAttributes(GCHandle securityDescr
               uint dwCreationDisposition,
               uint dwFlagsAndAttributes,
               IntPtr hTemplateFile);
-
-        [DllImport(PinvokeDllNames.WaitNamedPipeDllName, SetLastError = true, CharSet = CharSet.Unicode)]
```

Comment:
`WaitNamedPipeDllName` needs to be removed as well

Reply:
The list of consts contains many used names. We could clean up the list but all its lines are numbered. This is a work for a separate PR.

Reply:
I see.  Thanks.